README.md

```diff
@@ -7,14 +7,195 @@
+
+Get your `<token>` from [SolarWinds SaaS Free Trial](https://www.solarwinds.com/solarwinds-observability/registration).
+
+Start the app with tracing:
+```bash
+env OTEL_PHP_AUTOLOAD_ENABLED=true \
+    OTEL_SERVICE_NAME=php-example \
+    OTEL_TRACES_SAMPLER=solarwinds_http \
+    OTEL_PROPAGATORS=baggage,tracecontext,swotracestate,xtraceoptions \
+    OTEL_EXPERIMENTAL_RESPONSE_PROPAGATORS=xtrace,xtraceoptionsresponse \
+    OTEL_EXPORTER_OTLP_ENDPOINT=https://otel.collector.na-01.cloud.solarwinds.com:443 \
+    OTEL_EXPORTER_OTLP_HEADERS="Authorization=Bearer <token>" \
+    SW_APM_SERVICE_KEY=<token>:php-example \
+    php -S localhost:8080
+```
+
+Reload [http://localhost:8080/rolldice](http://localhost:8080/rolldice) and view traces in the SolarWinds Observability platform.
+
+<img width="616" alt="SWO" src="https://github.com/user-attachments/assets/ed312cc8-ebd7-4c4e-bce3-bac882843200" />
+
+---
+
+## Trace Context in Database Queries
+
+Install [OpenTelemetry SQL Commenter](https://packagist.org/packages/open-telemetry/opentelemetry-sqlcommenter) to propagate trace context to database queries:
+```bash
+composer require open-telemetry/opentelemetry-sqlcommenter
+```
+Supported DB libraries:
+- open-telemetry/opentelemetry-auto-mysqli
+- open-telemetry/opentelemetry-auto-pdo
+- open-telemetry/opentelemetry-auto-postgresql
+
+Enable context propagation:
+```bash
+OTEL_PHP_SQLCOMMENTER_CONTEXT_PROPAGATORS=tracecontext
+```
+Enable SQL comment attributes:
+```bash
+OTEL_PHP_SQLCOMMENTER_ATTRIBUTE=true
+```
+
+## Trace Context in Logs
+
+Install [OpenTelemetry Monolog logger](https://packagist.org/packages/open-telemetry/opentelemetry-logger-monolog) to propagate trace context to log records:
+```bash
+composer require open-telemetry/opentelemetry-logger-monolog
+```
+`service.name`, `traceId`, `spanId` & `flags` will be automatically added to OpenTelemetry logs.
+
+Example log output:
+```json
+{
+  ...
+  "resource":{"attributes":[{"key":"service.name","value":{"stringValue":"php-example"}}]},
+  ...
+  "logRecords":[{"timeUnixNano":"1762572315653380000","observedTimeUnixNano":"1762572315653559040","severityNumber":9,"severityText":"INFO","body":{"stringValue":"hello, otel"},"flags":1,"traceId":"1c52067371dfddaa6ed58c42d43d0a2f","spanId":"f33aaf87fc3ca8ab"}],
+  ...
+}
+```
+
+To add trace context to log messages, use OTEL SDK function `Context::getCurrent()`.
+
+Monolog example capturing the trace context with a [processor](https://seldaek.github.io/monolog/doc/01-usage.html#using-processors):
+```php
+use OpenTelemetry\API\Trace\Span;
+use OpenTelemetry\Context\Context;
+
+$logger->pushProcessor(function ($record) {
+  $spanContext = Span::fromContext(Context::getCurrent())->getContext();
+  $record['message'] .= ' trace_id='.$spanContext->getTraceId() . ' span_id=' . $spanContext->getSpanId() . ' trace_flags=' . ($spanContext->getTraceFlags() ? '01' : '00');
+  return $record;
+});
+```
+
+## Custom Transaction Name
```

Comment:
Looking at https://github.com/solarwinds/apm-php/tree/readme/src/API, there isn't a "[waitUntilReady](https://swicloud.atlassian.net/wiki/spaces/NIT/pages/3601400601/Swo-SDK+Application+Programming+Interface)" API call--I think we'd still need it, right?

For other languages we have a section describing custom instrumentation which talks about how to:
* use OTel API
* use our vendor specific API. 

see [apm-js](https://github.com/solarwinds/apm-js/blob/main/packages/solarwinds-apm/README.md#custom-instrumentation-and-metrics) and [apm-ruby](https://github.com/solarwinds/apm-ruby/blob/main/README.md#custom-instrumentation) for examples. Let's do that for apm-php too.

Reply:
...and a "forceFlush" call as well. I should audit if other APM libraries need to add this.

Reply:
forceFlush is not just for lambda, it applies to any deployment environment where we'd want batched telemetry to be exported (instead of lost) on application process exit.

I'm not getting the explanation on why waitUntilReady is no longer needed--are you saying that it's guaranteed at the time of a custom instrumentation "start span" call, that apm-php will have retrieved the sampling settings?

Reply:
Let's talk about this one in the afternoon meeting, with a follow-on PR to address.